src/qt/glcanvas.cpp

```diff
@@ -440,26 +494,33 @@ bool wxGLCanvas::SwapBuffers()
     return true;
 }
 
+bool wxGLCanvas::QtCanPaintWithoutActivePainter() const
+{
+    return true;
+}
+
 /* static */
-bool wxGLCanvas::ConvertWXAttrsToQtGL(const int *wxattrs, QGLFormat &format)
+bool wxGLCanvas::ConvertWXAttrsToQtGL(const wxGLAttributes &wxGLAttrs, const wxGLContextAttrs wxCtxAttrs, QGLFormat &format)
 {
-    if (!wxattrs)
-        return true;
+    const int *glattrs = wxGLAttrs.GetGLAttrs();
+    const int *ctxattrs = wxCtxAttrs.GetGLAttrs();
 
     // set default parameters to false
     format.setDoubleBuffer(false);
     format.setDepth(false);
     format.setAlpha(false);
```

Comment:
BTW, this seems unnecessary too, as it's disabled by default according to [the docs](https://doc.qt.io/qt-5/qglformat.html#setAlpha), so maybe remove it too?

```suggestion
```

Reply:
Since we call `format.setAlpha(true);` later, I think it's better to set it to false explicitly here.